src/docker/constants.ts

```diff
@@ -3,7 +3,7 @@
  *
  * Must match the default value in the "confluent.docker.localSchemaRegistryImageTag" configuration in package.json.
  */
-export const DEFAULT_SCHEMA_REGISTRY_TAG = "latest";
+export const DEFAULT_SCHEMA_REGISTRY_TAG = "7.9.0";
```

Comment:
Ah sorry @eguicf, I should have deleted this as part of the settings refactor [PR](https://github.com/confluentinc/vscode/pull/2186). Nothing currently uses this variable, so do you mind deleting it altogether?